openml_OS/models/Author.php

```diff
@@ -1,10 +1,12 @@
 <?php
+// deprecated, pleas don't use.
```

Comment:
Typo. Plus, why do we still need it if we (the only users?) will not use anymore?